velox/connectors/hive/HiveDataSink.cpp

```diff
@@ -506,7 +506,7 @@ std::pair<std::string, std::string> HiveDataSink::getWriterFileNames(
         "{}_{}_{}",
         connectorQueryCtx_->taskId(),
         connectorQueryCtx_->driverId(),
-        isCommitRequired() ? "0" : makeUuid());
+        makeUuid());
```

Comment:
Would be nice to add a comment here to explain why we include a UUID. 

BTW, would it make sense to add plan Node ID for debuggability? taskId.driverId.planNodeId.uuid

Reply:
Hi Masha, agree, let me add the plan node ID for debuggability. Recently see the need for good debuggability.

Reply:
Do we need the uuid if we have the `planNodeId` ?

Reply:
> Do we need the uuid if we have the `planNodeId` ?

You're right, if we have planNodeId we do not need the uuid part, given all the cases we've seen so far. Adding uuid randomness now just to keep the approach consistent with PrestoSpark Java, and to avoid future problem, like https://github.com/prestodb/presto/blob/3fbbb677da4c9dc2386379d6a9d523fc2ae3c1c8/presto-hive/src/main/java/com/facebook/presto/hive/HiveWriterFactory.java#L356